images/julia/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM julia:latest
+
+COPY . /app
+
+RUN apt-get update && apt-get install -y gcc apt-utils unzip make libhttp-parser-dev
+RUN julia /app/deps.jl
+
+CMD julia /app/app.jl
+EXPOSE 8080
```

Comment:
@mbesancon can you expose 3000 instead 8080, let's keep same port like the other Dockerfile of fx/images

Reply:
Yep, I'll change that tomorrow